Formula/grpc.rb

```diff
@@ -1,8 +1,8 @@
 class Grpc < Formula
```

Comment:
That's a good point. Actually I have submitted our grpc formula to the [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/grpc.rb) repository and there it seems to be automatically pushed forward each release (i.e. I didn't do anything and the formula got updated to 1.2.0). There it also depends on `protobuf` formula not `google-protobuf`, and people seem to be having no issue using that formula. In fact it's easier for user as they don't have to tap `grpc` anymore. 

So is there a reason we need to maintain this repository?